lib/screen/detail/artwork_detail_page.dart

```diff
@@ -744,7 +744,7 @@ class _ArtworkDetailPageState extends State<ArtworkDetailPage>
             onTap: () async {
               await injector<ConfigurationService>()
                   .updateTempStorageHiddenTokenIDs([asset.id], !isHidden);
-              unawaited(injector<SettingsDataService>().backup());
+              unawaited(injector<SettingsDataService>().backupUserSettings());
```

Comment:
here we don't have to backup all user setting, just need to update hidden artwork setting.